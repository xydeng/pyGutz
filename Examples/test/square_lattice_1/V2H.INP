NT= 1
   1   2   1   2       8.000000000000       0.000000000000
   2   1   2   1       8.000000000000       0.000000000000
