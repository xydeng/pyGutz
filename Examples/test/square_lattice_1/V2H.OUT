NT=  1
  1  2  1  2        8.0000000000      0.0000000000
  2  1  2  1        8.0000000000      0.0000000000
